components/TARGET_PSA/services/crypto/COMPONENT_SPE/psa_crypto_partition.c

```diff
@@ -1479,7 +1479,7 @@ static void psa_entropy_operation(void)
             mbedtls_free(seed);
 #else
             status = PSA_ERROR_NOT_SUPPORTED;
-#endif /* MBEDTLS_ENTROPY_NV_SEED && MBEDTLS_PSA_HAS_ITS_IO*/
+#endif /* MBEDTLS_PSA_INJECT_ENTROPY */
```

Comment:
I followed the change in another PSA file.

@Patater @k-stachowiak Is this correct?

Reply:
Matches change in features/mbedtls/mbed-crypto/platform/TARGET_PSA/COMPONENT_PSA_SRV_IMPL/psa_crypto.c

Reply:
It makes sense to align the macros guarding APIs use with the macros guarding its definition. I haven't been following the development of this feature, but to me, this change looks correct.

Reply:
I confirm that the compile-time guard for `mbedtls_psa_inject_entropy` has changed to `MBEDTLS_PSA_INJECT_ENTROPY`. I haven't reviewed the rest of the PR.